hack/update-codegen.sh

```diff
@@ -37,10 +42,15 @@ ${GOPATH}/bin/deepcopy-gen \
   -i github.com/tektoncd/pipeline/pkg/apis/config
 
 # Knative Injection
+bash ${REPO_ROOT_DIR}/hack/generate-knative.sh "injection" \
+  github.com/tektoncd/pipeline/pkg/client/resource github.com/tektoncd/pipeline/pkg/apis \
+  "resource:v1alpha1" \
+  --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt
 bash ${REPO_ROOT_DIR}/hack/generate-knative.sh "injection" \
   github.com/tektoncd/pipeline/pkg/client github.com/tektoncd/pipeline/pkg/apis \
   "pipeline:v1alpha1,v1alpha2" \
   --go-header-file ${REPO_ROOT_DIR}/hack/boilerplate/boilerplate.go.txt
+GOFLAGS="${OLDGOFLAGS}"
```

Comment:
is it possible to add some comments around the enw lines in this script explaining what they do? and/or explain in the commit message? iirc from the previous review, it was not obvious what you had to do to make this work without a different apigroup, might be useful to record how this ended up working

Reply:
ah this comes from https://github.com/tektoncd/pipeline/pull/1783.. forgot to mention that :joy: :sweat:

Reply:
I need to update comments on the `bash …` calls though, to explain why there is 2 indeed

Reply:
kk! happy for that to be in follow up PR :)